api/tests/core/mails/transactional/pro/offer_validation_to_pro_test.py

```diff
@@ -19,10 +19,13 @@
 
 
 class SendinblueSendOfferValidationTest:
-    def test_get_validation_approval_correct_email_metadata(self):
+    @pytest.mark.parametrize("offer_is_digital", [True, False])
+    def test_get_validation_approval_correct_email_metadata(self, offer_is_digital):
```

Comment:
Suggestion : 
```suggestion
    @pytest.mark.parametrize("factory_class", [offers_factories.DigitalOfferFactory, offers_factories.OfferFactory])
    def test_get_validation_approval_correct_email_metadata(self, factory_class):
```
Et donc le bloc en-dessous devient : 
```
    offer = factory_class(name="Ma petite offre", venue__name="Mon stade")
```
C'est une question de goût, mais j'aime bien, ça retire des `if` :)